Jellyfin.Server.Implementations/Security/AuthorizationContext.cs

```diff
@@ -185,9 +185,21 @@ private async Task<AuthorizationInfo> GetAuthorization(HttpContext httpReq)
                     authInfo.IsAuthenticated = true;
                     authInfo.Client = key.Name;
                     authInfo.Token = key.AccessToken;
-                    authInfo.DeviceId = string.Empty;
-                    authInfo.Device = string.Empty;
-                    authInfo.Version = string.Empty;
+                    if(string.IsNullOrWhiteSpace(authInfo.DeviceId))
+                    {
+                        authInfo.DeviceId = string.Empty;
+                    }
+
+                    if(string.IsNullOrWhiteSpace(authInfo.Device))
+                    {
+                        authInfo.Device = string.Empty;
+                    }
+
+                    if(string.IsNullOrWhiteSpace(authInfo.Version))
```

Comment:
The suggestion tool isn't working for me, but there should be a single space between `if` and `(`, like so: `if (string.IsNull...`